turbo.json

```diff
@@ -1,6 +1,10 @@
 {
   "$schema": "https://turborepo.com/schema.json",
   "tasks": {
+    "build": {
+      "dependsOn": ["^build"],
+      "outputs": ["dist/**"]
+    },
```

Comment:
Adding the `build` command as a task so `turbo` can cache the build output, eg; the `dist` directory when running `tsdown`